src/Relude/Functor/Reexport.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP    #-}
```

Comment:
Hmm, for some reason `stylish-haskell` hasn't been applied to this file 🤔

Reply:
Good spot, looking into this. Seems like I didn't run the command properly :|